tests/main/proxy/task.yaml

```diff
@@ -21,12 +14,11 @@ execute: |
     echo "Check that the commands go through the proxy"
     snap find test-snapd-tools | MATCH test-snapd-tools
 
-    # ensure squid wrote the access.log
-    systemctl restart squid
+    # check unit outout
     for _ in $(seq 10); do
```

Comment:
This line replace the whole for, and it uses the cursor to search on the log for the current test.
check_journalctl_log 'CONNECT api.snapcraft.io' -u tinyproxy

Reply:
Nice! Thank you.